.changeset/heavy-points-marry.md

```diff
@@ -0,0 +1,43 @@
+---
+'@primer/react': major
+---
+
+![NewLabels](https://user-images.githubusercontent.com/2313998/155384037-8a56c943-4cff-4349-925b-dc28247e3491.png)
+
+The Label component has been updated to align with [Primer ViewComponents' Label component](https://primer.style/view-components/components/label). Now the React component is consistent ViewComponents and the intent of the design team.
```

Comment:
```suggestion
The Label component has been updated to be visually consistent with its counterpart in [Primer ViewComponents' Label component](https://primer.style/view-components/components/label).
```

Reply:
Great suggestion. We also updated the API to be more like PVC, so I'll also mention that.